public/dev/js/science_portal_session.js

```diff
@@ -337,6 +338,48 @@
       })
     }
 
+
+    function renewSession(sessionID) {
+      var _formData = new FormData();
+      _formData.append("action", "renew")
+
+      Promise.resolve(renewSessionAjax(_selfPortalSess.sessionServiceURL + "/" + sessionID, sessionID, _formData))
+        .then(function(sessionID) {
+          trigger(_selfPortalSess, cadc.web.science.portal.session.events.onSessionActionDone, sessionID)
+        })
+        .catch(function(message) {
+          // get session list failed in a way that can't allow page to continue
+          trigger(_selfPortalSess, cadc.web.science.portal.session.events.onSessionActionError, message)
+        })
+    }
+
+    function renewSessionAjax(serviceURL, sessionID, sessionAction) {
+      return new Promise(function (resolve, reject) {
+        var request = new XMLHttpRequest()
+
+        // "load" is the XMLHttpRequest "finished" event
+        request.addEventListener(
+          "load",
+          function () {
+            if (request.status === 200) {
+              resolve(sessionID)
+            } else {
+              reject(request)
+            }
+          },
+          false
+        )
+        // withCredentials enables cookies to be sent
+        // Note: SameSite cookie header isn't set with this method,
```

Comment:
Is this comment still relevant?  We set `SameSite: None` on our cookies now.

Reply:
It's not relelvant, agreed. Removed 2 comments from this file.